README.md

```diff
@@ -188,113 +252,166 @@ The following CSS:
 }
 ```
 
-will be processed to:
+This will be transformed to:
 
 ```css
 .box-foo {
 	width: 150px;
+}
 
-	.box-bar {
-		width: 150px;
-	}
-
-	@media (max-width: 800px) {
-		.box-bar {
-			width: 300px;
-		}
-	}
+.box-foo .box-bar {
+	width: 150px;
 }
 
 @media (max-width: 800px) {
 	.box-foo {
 		width: 300px;
 	}
+	
+	.box-foo .box-bar {
+		width: 300px;
+	}
 }
 ```
 
+### `calc()`
```

Comment:
I'm just saying it's especially nice when you're dealing with variables. To reuse a Stylus example, check out this:

```css
#logo {
   position: absolute;
   top: 50%;
   left: 50%;
   --w: 150px;
   --h: 80px;
   width: var(--w);
   height: var(--h);
   margin-left: calc(var(--w) / -2);
   margin-top: calc(var(--h) / -2);
}
```

It's not so much whether it's useful in general (I agree it is good regardless), but variables change the equation a *lot*.